pkgs/by-name/ws/wsjtz/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  wsjtx,
+  fetchzip,
+  lib,
+}:
+
+wsjtx.overrideAttrs (old: rec {
```

Comment:
Why the rec?

Reply:
Don't I need it because I'm defining `version` and using it in the same block?

Reply:
Oh never mind.

Reply:
No worries :3 I'm super new to nix so I wanted to make sure I wasn't missing something